content/zh-cn/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -18,7 +18,7 @@ weight: 45
 <!--
 In Kubernetes, _namespaces_ provides a mechanism for isolating groups of resources within a single cluster. Names of resources need to be unique within a namespace, but not across namespaces. Namespace-based scoping is applicable only for namespaced {{< glossary_tooltip text="objects" term_id="object" >}} _(e.g. Deployments, Services, etc)_ and not for cluster-wide objects _(e.g. StorageClass, Nodes, PersistentVolumes, etc)_.
 -->
-在 Kubernetes 中，**名字空间（Namespace）**提供一种机制，将同一集群中的资源划分为相互隔离的组。
+在 Kubernetes 中，*名字空间（Namespace）* 提供一种机制，将同一集群中的资源划分为相互隔离的组。
```

Comment:
```suggestion
在 Kubernetes 中，**名字空间（Namespace）** 提供一种机制，将同一集群中的资源划分为相互隔离的组。
```

Reply:
Thanks so much for the review! @windsonsea 
I will commit the new updates to the PR commit.